homeassistant/components/ovo_energy/config_flow.py

```diff
@@ -0,0 +1,66 @@
+"""Config flow to configure the OVO Energy integration."""
+import logging
+
+import aiohttp
+from ovoenergy.ovoenergy import OVOEnergy
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.config_entries import ConfigFlow
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
+
+from .const import CONF_ACCOUNT_ID, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+@config_entries.HANDLERS.register(DOMAIN)
+class OVOEnergyFlowHandler(ConfigFlow):
+    """Handle a OVO Energy config flow."""
+
+    VERSION = 1
+    CONNECTION_CLASS = config_entries.CONN_CLASS_CLOUD_POLL
+
+    def __init__(self):
+        """Initialize OVO Energy flow."""
+
+    async def _show_setup_form(self, errors=None):
+        """Show the setup form to the user."""
+        return self.async_show_form(
```

Comment:
Please inline this in the user step instead. There's no need to make this a method.